src/ckms.rs

```diff
@@ -64,13 +67,25 @@ impl<T> AddAssign for CKMS<T>
             (Some(x), None) => Some(x),
             (Some(x), Some(y)) => Some(x.add(y)),
         };
+        self.cma = match (self.cma, rhs.cma) {
```

Comment:
No worries! 

To start, `cma` is 'cumulative moving average'. If we back up in the definition a little bit the function is `fn add_assign(&mut self, rhs: CKMS<T>)`. We're modifying `self` and consuming `rhs`. What we're doing here in line 70 is  absorbing the value of `rhs.cma` into `self.cma` in the same fashion as sum.